src/shared/utils/getJwtMaxAge.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ *  ## JwtPayload
+ *  @type [exp] - 만료 시각
+ *  @type [iat] - 발급 시각
+ */
+type JwtPayload = {
+  exp?: number;
+  iat?: number;
+};
+
+/**
+ * ## getJwtMaxAge
+ * JWT 토큰의 payload를 디코딩하여
+ * 현재 시점 기준으로 남아있는 유효 시간을 초 단위로 반환합니다.
+ *
+ * @param token - JWT 문자열 (accessToken 또는 refreshToken)
+ * @returns 토큰 유효시간, exp가 없는 경우 undefined
+ */
+export const getJwtMaxAge = (token: string): number | undefined => {
+  try {
+    const payloadBase64 = token.split('.')[1];
+    const payloadJson = Buffer.from(payloadBase64, 'base64').toString('utf-8');
+    const payload = JSON.parse(payloadJson) as JwtPayload;
+
+    if (!payload.exp) {
+      return undefined;
+    }
```

Comment:
숫자인지도 확인하면 좋지 않을까요?
`payload.exp`가 존재하더라도 숫자가 아니라면 계산 로직에서 오류가 발생할 것 같습니다!


```suggestion
    if (!payload.exp || typeof payload.exp !== 'number') {
      return undefined;
    }
```